source/simulator/melt.cc

```diff
@@ -60,6 +60,29 @@ namespace aspect
 
   namespace Assemblers
   {
+    namespace
+    {
+      template<int dim>
+      bool
+      is_stokes_component (const Introspection<dim> &introspection,
```

Comment:
I think that would have been fine, because both pressures are inside the "Stokes part" (it is called assemble_stokes after all).